test/commands/BeautifyCommand.spec.ts

```diff
@@ -138,6 +150,33 @@ describe("BeautifyCommand", () => {
             expect(json).toMatchSnapshot("json");
           });
       });
+      test("should throw an error with invalid json", () => {
+        const command = new CustomCommand();
+        const thenCb = jest.fn();
+        const catchCb = jest.fn();
+        return command
+          .beautify({
+            args: [],
+            configJson: `{"JavaScript": {"beautifiers": ["ESLint"],"quotes": "double"`,
+            filePath: "test/fixtures/test1.js",
+            language: "JavaScript",
+          })
+          .then(thenCb)
+          .catch(catchCb)
+          .then(() => {
+            expect(thenCb).not.toBeCalled();
+            expect(catchCb).toHaveBeenCalled();
+            expect(catchCb.mock.calls).toHaveLength(1);
+            expect(catchCb.mock.calls[0]).toHaveLength(1);
+            expect(catchCb).toHaveProperty(
+              ["mock", "calls", 0, 0, "message"],
+              "Unexpected end of JSON input"
+            );
+            const json = command.toJSON();
+            expect(json.exitCode).toBe(1);
+            expect(json).toMatchSnapshot("json");
```

Comment:
I suppose we don't really need the snapshot since we have the `expect` from lines 171-173. I left it in for my other tests, too.